lib/evaluation/handlers/sameSubjectEvaluationHandler.ts

```diff
@@ -16,25 +16,8 @@ import { HandlerCheckResult } from '../handlerCheckResult';
 import { AbstractEvaluationHandler } from './abstractEvaluationHandler';
 
 export class SameSubjectEvaluationHandler extends AbstractEvaluationHandler {
-  private pDefinition: PresentationDefinition = { id: '', input_descriptors: [] };
-  private vPresentation: VerifiablePresentation = {
-    '@context': [],
-    holder: '',
-    presentation_submission: {
-      id: '',
-      definition_id: '',
-      descriptor_map: [],
-    },
-    proof: {
-      type: '',
-      created: '',
-      proofPurpose: '',
-      verificationMethod: '',
-      jws: '',
-    },
-    type: '',
-    verifiableCredential: [],
-  };
+  private pDefinition: PresentationDefinition | undefined;
```

Comment:
What is the need for the undefined? The definition and presentation should be available for this handler to make sense right?